presentation/src/main/java/com/shifthackz/aisdv1/presentation/activity/AiStableDiffusionActivity.kt

```diff
@@ -85,7 +85,9 @@ class AiStableDiffusionActivity : AppCompatActivity() {
 
             LaunchedEffect(backStackEntry) {
                 if (!viewModel.state.value.isShowSplash) return@LaunchedEffect
-                if (backStackEntry?.destination?.route != Constants.ROUTE_SPLASH) {
+                backStackEntry?.let { entry ->
+                    val route = entry.destination.route ?: ""
```

Comment:
What is the value of `route` in this case?

Reply:
route=com.shifthackz.aisdv1.presentation.navigation.NavigationRoute.Splash
route=com.shifthackz.aisdv1.presentation.navigation.NavigationRoute.Onboarding/{source}
route=com.shifthackz.aisdv1.presentation.navigation.NavigationRoute.GalleryDetail/{itemId}